src/pages/iou/IOUCurrencySelection.js

```diff
@@ -43,10 +45,19 @@ const propTypes = {
         currency: PropTypes.string,
     }),
 
+    /** The report on which the request is initiated on */
+    report: reportPropTypes,
+
+    /** Any errors associated with an attempt to create a chat */
+    // eslint-disable-next-line react/forbid-prop-types
+    errors: PropTypes.object,
```

Comment:
Where is this prop coming from exactly?

Reply:
I don't see anything passing an `errors` object.

Reply:
Hey @marcaaron, not just this. I noticed that in 3 out of 4 occurrences where `shouldHideComposer` is used. We are using `props.errors` when there is no prop `errors`. Here is a list -

1. ReportScreen - for hiding EmojiPickers  https://github.com/Expensify/App/blob/058978e356bd266ee803a92467c3592565ad0c0c/src/pages/home/ReportScreen.js#L164
2. MoneyRequestAmountPage - for dismissing modal 
    https://github.com/Expensify/App/blob/058978e356bd266ee803a92467c3592565ad0c0c/src/pages/iou/steps/MoneyRequestAmountPage.js#L213

3. This one.

The only correct occurrence where errors are actually populated is in `ReportFooter` to hide the composer. The `errors` used there are -
```javascript
lodashGet(report, 'errorFields.addWorkspaceRoom') || lodashGet(report, 'errorFields.createChat');
```

I will be happy to fix this if you want.

Reply:
Awesome, that would be great. I'll create an issue for this.

Reply:
https://github.com/Expensify/App/issues/23373